.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
IMO having this in git is fine, because a lot of people seem to use vscode, but I want to know what @PurpleMyst thinks about this.

Reply:
I don't think editor-specific files should go into a repository, and I prefer stuff like editorconfig files where possible; neither me nor you use VScode, however it doesn't really matter to me, it can be left in if it's of use to somebody contributing code. :)